skipruntime-ts/wasm/package.json

```diff
@@ -3,7 +3,10 @@
   "version": "0.0.10",
   "type": "module",
   "exports": {
-    ".": "./dist/src/skip-runtime.js"
+    ".": {
+      "node": "./dist/src/node.js",
+      "default": "./dist/src/browser.js"
```

Comment:
Interesting. FMU, where do these "node" and "default" keys come from? Is it something built in to npm?